src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -511,18 +514,16 @@ private static IdTitleViewModel BuildPublicationViewModel(PublicationMethodology
             return new IdTitleViewModel(publication.Id, publication.Title);
         }
 
-        private async Task<Either<ActionResult, Unit>> ValidateMethodologySlugUniqueForUpdate(
-            MethodologyVersion methodologyVersion, string slug)
+        private async Task<Either<ActionResult, Unit>> ValidateMethodologySlugUnused(string slug)
         {
-            if (await _context
-                    .Methodologies
-                    .AsQueryable()
-                    .AnyAsync(p => p.Slug == slug && p.Id != methodologyVersion.MethodologyId))
+            var methodologyVersion = await _methodologyVersionRepository.GetLatestPublishedVersionBySlug(slug);
+
+            if (methodologyVersion != null)
             {
                 return ValidationActionResult(SlugNotUnique);
```

Comment:
Maybe I'm missing something obvious, but as we create the new methodology on `PublicationMethodologyPagej.tsx` via a button rather than a form, there is no `useFormSubmit` to pass the error mapper to.

Reply:
As discussed in Slack, realised that methodology creation is done via a button click so this currently isn't applicable.